src/panels/lovelace/cards/hui-plant-status-card.js

```diff
@@ -2,7 +2,21 @@ import "../../../cards/ha-plant-card";
 
 import LegacyWrapperCard from "./hui-legacy-wrapper-card";
 
+export const Config = {
```

Comment:
Not how you define an interface.

Reply:
Do you suggest I do it in a TS file then? This is what I found when googling for just JS interfaces as they don’t really exist in JS.